tools/ci_connector_ops/setup.py

```diff
@@ -45,7 +45,7 @@ def local_pkg(name: str) -> str:
 ]
 
 PIPELINES_REQUIREMENTS = [
-    "dagger-io==0.5.4",
+    "dagger-io==0.6.4",
```

Comment:
This release is coming out in just a few hours. The minimum needed for these changes is `0.6.0`.